src/pykeen/datasets/ogb.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+
+"""Load the OGB datasets.
+
+Run with python -m pykeen.datasets.ogb
+"""
+
+from typing import ClassVar, Optional
+
+import numpy as np
+
+from .base import LazyDataSet
+from ..triples import TriplesFactory
+
+__all__ = [
+    'OGBLoader',
+    'OGBBioKG',
+    'OGBWikiKG',
+]
+
+
+class OGBLoader(LazyDataSet):
+    """Load from the Open Graph Benchmark (OGB)."""
+
+    #: The name of the dataset to download
+    name: ClassVar[str]
+
+    def __init__(self, cache_root: Optional[str] = None, create_inverse_triples: bool = False):
+        self.cache_root = self._help_cache(cache_root)
+        self.create_inverse_triples = create_inverse_triples
+
+    def _load(self) -> None:
+        try:
+            from ogb.linkproppred import LinkPropPredDataset
+        except ImportError as e:
+            raise ModuleNotFoundError(
+                f'Need to `pip install ogb` to use pykeen.datasets.{self.__class__.__name__}.',
+            ) from e
+
+        self.dataset = LinkPropPredDataset(name=self.name, root=self.cache_root)
```

Comment:
Do we need the reference to the dataset anywhere? If not, just using a local variable allows freeing memory afterwards.

Reply:
No, good point. Fixed in ad70d29c